python/doc/theory/numerical_methods/rosenblatt_transformation.rst

```diff
@@ -3,49 +3,46 @@
-| One way to evaluate the probability content of the event
-  :math:`\cD_f`:
-
-  .. math::
+The Rosenblatt transformation is an :doc:`isoprobabilistic_transformation` which is used under the following 
+context :the input random vector is :math:`\vect{X}` with marginal cumulative density functions  :math:`F_i` and 
+copula :math:`C`. Nothing special is assumed about the copula.
+
+Let :math:`\vect{d}` be a  deterministic vector, let :math:`g(\vect{X}\,,\,\vect{d})` be the limit state function 
+of the model and let :math:`\cD_f = \{\vect{X} \in \Rset^n \,/ \, g(\vect{X}\,,\,\vect{d}) \le 0\}` be an event 
+which probability :math:`P_f` is defined as: 
+
+.. math::
     :label: PfX2
 
       P_f = \Prob{g(\vect{X}\,,\,\vect{d})\leq 0}=   \int_{\cD_f}  \pdf\, d\vect{x}
 
-  is to use the Rosenblatt transformation :math:`T` which is a
-  diffeomorphism from :math:`\supp{\vect{X}}` into the standard space
-  :math:`\Rset^n`, where distributions are normal, with zero mean, unit
-  variance and unit correlation matrix (which is equivalent in that
-  normal case to independent components).
+is to use the Rosenblatt transformation :math:`T` which is a
+diffeomorphism from the support of the distribution :math:`f_{\vect{X}}` into the standard space
+:math:`\Rset^n`, where distributions are normal, with zero mean, unit
+variance and unit correlation matrix (which is equivalent in that
+normal case to independent components).
 
-| Let us recall some definitions.
-| The *cumulative distribution function* :math:`F_{1,k}` of the
-  :math:`k`-dimensional random vector :math:`(X_1, \dots, X_k)` is
-  defined by its marginal distributions :math:`F_i` and the copula
-  :math:`C_{1,k}` through the relation:
+Let us recall some definitions.
+The *cumulative distribution function* :math:`F_{1,k}` of the
+:math:`k`-dimensional random vector :math:`(X_1, \dots, X_k)` is
+defined by its marginal distributions :math:`F_i` and the copula
+:math:`C_{1,k}` through the relation:
 
-  .. math:: F_{1,k}(x_1,\dots, x_k) = C_{1,k}(F_1(x_1),\dots, F_k(x_k))
+.. math:: F_{1,k}(x_1,\dots, x_k) = C_{1,k}(F_1(x_1),\dots, F_k(x_k))
 
-   with
+with
 
-  .. math::
+.. math::
     :label: subCopula
 
       C_{1,k}(u_1, \dots, u_k) = C(u_1, \dots, u_k, 1, \dots, 1)
 
 The *cumulative distribution function* of the conditional variable
 :math:`X_k|X_1, \dots, X_{k-1}` is defined by:
 
-.. math:: F_{k|1, \dots, k-1} (x_k|x_1, \dots, x_{k-1})   =  \displaystyle \frac{\partial^{k-1} F_{1,k}(x_1, \dots, x_k)}{\partial x_1 \dots \partial x_{k-1}} /\frac{\partial^{k-1} F_{1,k-1}(x_1, \dots, x_{k-1})} {\partial x_1 \dots \partial x_{k-1}}
+.. math:: 
+
+    F_{k|1, \dots, k-1} (x_k|x_1, \dots, x_{k-1})   =  \displaystyle \frac{\partial^{k-1} F_{1,k}(x_1, \dots, x_k)}{\partial x_1 \dots \partial x_{k-1}} /\frac{\partial^{k-1} F_{1,k-1}(x_1, \dots, x_{k-1})} {\partial x_1 \dots \partial x_{k-1}}
 
 **Rosenblatt transformation**: Let :math:`\vect{X}` in :math:`\Rset^n`
```

Comment:
This will create a proper sub-section:
```suggestion
Rosenblatt transformation
---------------------------------------

Let :math:`\vect{X}` in :math:`\Rset^n`
```

Reply:
ok